### ISSUE TITLE
aspectRatio fails when there is a geo mark and no projection

### ISSUE THREAD

**Issue by Fil** [observablehq/plot#1507]
```js
values = d3.cross(d3.range(10), d3.range(10)).map(([i, j]) => i * j);
```
```js
Plot.plot({
  aspectRatio: true,
  inset: 40,
  x: { ticks: 10 },
  marks: [
    Plot.dot(values, {
      x: (d, i) => i % 10 + 0.5,
      y: (d, i) => Math.floor(i / 10) + 0.5,
      fill: (d) => d
    }),
    Plot.geo([])
  ]
})
```

Note: in that case aspectRatio is not completely ignored, but defaults to the golden ratio.

**Comment by mbostock** [comment]
Perhaps this is fixed by #1468?

**Comment by Fil** [comment]
Yes it is!
